src/core/reference/src/runtime/reference/convert.cpp

```diff
@@ -231,6 +283,42 @@ void convert<float16, int8_t>(const float16* arg, int8_t* out, size_t count) {
     convert_impl(arg, out, count);
 }
 
+void convert_from_f32_to_f16_with_clamp(const float* arg, float16* out, size_t count) {
+    #    if defined(OPENVINO_ARCH_X86) || defined(OPENVINO_ARCH_X86_64)
```

Comment:
suggest moving this compile time condition outside of the function

Reply:
done